ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-ec-4.13.yaml

```diff
@@ -159,6 +163,20 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test-disconnected
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private
+- as: aws-ipi-ovn-hypershift-p1-f7
```

Comment:
@LiangquanLi930 Per `aws-ipi-ovn-hypershift-p1-f7`, I do not know it is hosted cluster or mgmt cluster testing although I know it for hosted cluster from workflow name `cucushift-installer-rehearse-aws-ipi-ovn-hypershift-guest`.
so, better to add keyword into test name to indicate it, like `guest`

Reply:
Sure, Thanks for your reminder^^

Reply:
@kuiwang02 For the hypershift upgrade we need two pipelines, CP upgrade and nodepool upgrade, So I think we should name it like this 
`aws-ipi-ovn-hypershift-cp-p1-f7` and `aws-ipi-ovn-hypershift-nodepool-p1-f7`
Do you have some idea?
cc @heliubj18

Reply:
i would prefer something like `aws-ipi-ovn-hypershift-cp-upgrade-p1-f7` and `aws-ipi-ovn-hypershift-nodepool-upgrade-p1-f7`

Reply:
`aws-ipi-ovn-hypershift-cp-p1-f7`  and `aws-ipi-ovn-hypershift-nodepool-p1-f7` We don't need to add upgrade

Reply:
@LiangquanLi930 what is the CP ugprade? does it mean hypershift mgtm cluster upgrade?
what is the nodepool upgrade? does it mean hypershift hosted cluster upgrade?

@heliubj18 I prefer no to have word `upgrade` in test name because `upgrade` is in file name which is finally in job name. you could see other upgrade test name also has no `upgrade`

Reply:
got it

Reply:
@kuiwang02 let's use the one from @LiangquanLi930

Reply:
@kuiwang02 cp upgrade is hypershift controlplane upgrade

Reply:
It is not about mgmt or hosted cluster. They are all hosted cluster upgrade. One is the controlplane upgrade and another is nodepool upgrade.

Reply:
got it. thanks

Reply:
@LiangquanLi930 @heliubj18 
I have no other comment for current code besides the comment on test name.